lib/commands/start.js

```diff
@@ -64,7 +63,8 @@ class StartCommand extends Command {
                         const isInstall = process.argv[2] === 'install';
                         let adminUrl = instance.config.get('admin.url') || instance.config.get('url');
 
-                        adminUrl = url.resolve(adminUrl, '/ghost/');
+                        // Strip the trailing slash and add the admin path
+                        adminUrl = adminUrl.replace(/\/$/,'') + '/ghost/';
```

Comment:
just to keep things consistent - can this be a template string?

Reply:
Yep, done 😃